docker/.python-version

```diff
@@ -5,4 +5,4 @@
 3.7.13
 3.8.13
 3.9.11
-3.11-dev
+3.11.0rc1
```

Comment:
do we care here? `-dev` should get us latest rc no?

also this change needs to happen in it's own PR since CircleCI uses the _published_ image, which won't get published until this change is merged into `1.x` branch

Reply:
Sounds good, does this mean that I can just revert this back to `3.11-dev` and wait until 3.11 is officially released?

Reply:
yes, it should be ok to revert it